Jellyfin.Api/Controllers/SessionController.cs

```diff
@@ -294,10 +294,7 @@ await _sessionManager.SendGeneralCommand(currentSession.Id, sessionId, generalCo
         {
             var currentSession = await RequestHelpers.GetSession(_sessionManager, _userManager, HttpContext).ConfigureAwait(false);
 
-            if (command == null)
-            {
-                throw new ArgumentException("Request body may not be null");
-            }
+            ArgumentNullException.ThrowIfNull(command, "Request body may not be null");
```

Comment:
This is wrong https://learn.microsoft.com/en-us/dotnet/api/system.argumentnullexception.throwifnull?view=net-7.0#system-argumentnullexception-throwifnull(system-object-system-string)

Reply:
Fixed